RussianFilter/sections/antiadblock.txt

```diff
@@ -37,9 +37,9 @@ zannn.top,innal.top$$script[tag-content="\x66\x3D\x27\x2F\x2F\x72"][max-length="
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38088
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39666
 ! https://github.com/AdguardTeam/AdguardFilters/issues/40041
-!+ PLATFORM(ext_chromium, ext_edge)
-stb.ua,novy.tv,ictv.ua,teleportal.ua#%#AG_abortOnPropertyRead('STRICT_ADBLOCK');
-!#if (adguard_ext_firefox || adguard_ext_opera || adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)
+! https://github.com/AdguardTeam/AdguardFilters/issues/41277
+||static.adtelligent.com/static/jsvpaid.js^$empty,important,domain=stb.ua|novy.tv|ictv.ua|teleportal.ua
```

Comment:
In case of Microsoft Edge - `$empty` rule doesn't work, so I think it's necessary to add:
```
!+ PLATFORM(ext_edge)
@@||static.adtelligent.com/static/jsvpaid.js^$domain=stb.ua|novy.tv|ictv.ua|teleportal.ua
```
By the way, I didn't check it with content blockers, is not needed also for iOS, Safari?

Reply:
We have rules for content blockers, so I added edge platform to them.